client/landing/gutenboarding/onboarding-block/create-site/index.tsx

```diff
@@ -14,16 +14,16 @@ import './style.scss';
 
 /* eslint-disable wpcalypso/jsx-classname-namespace */
 const CreateSite: FunctionComponent< {} > = () => {
-	const { __: NO__ } = useI18n();
+	const { __ } = useI18n();
 	const shouldTriggerCreate = useNewQueryParam();
 	const [ shouldCreateAndRedirect, setCreateAndRedirect ] = React.useState( false );
 
 	// Some very rudimentary progress illusions
 
 	const progressSteps = [
-		NO__( 'Building your site' ),
-		NO__( 'Getting your domain' ),
-		NO__( 'Applying design' ),
+		__( 'Building your site' ),
+		__( 'Getting your domain' ),
+		__( 'Applying design' ),
 	];
 
 	return (
```

Comment:
Inside this return there's this that needs translation function:

```
Step ${ index + 1 } of ${ progressSteps.length }
```

Reply:
ffad542580cc9b3228b9679f51c9a85012bec118